servers/media-server/src/server/connector/transports/mod.rs

```diff
@@ -1,17 +1,34 @@
-use std::io;
+use std::{fmt::Display, io};
 
 use async_trait::async_trait;
+use clap::ValueEnum;
 use prost::Message;
 
 pub mod nats;
+pub mod http;
 
 #[derive(Debug, PartialEq, Eq)]
 pub enum ParseURIError {
     InvalidURI,
 }
 
+#[derive(Debug, ValueEnum, Clone)]
+pub enum Format {
+    Json,
+    Protobuf,
+}
+
+impl Display for Format {
```

Comment:
maybe we should switch to [derive_more](https://crates.io/crates/derive_more) for simpler